docs/api/app.md

```diff
@@ -589,7 +624,7 @@ Hides all application windows without minimizing them.
 
 ### `app.isHidden()` _macOS_
 
-Returns `boolean` - `true` if the application—including all of its windows—is hidden (e.g. with `Command-H`), `false` otherwise.
+Returns `boolean` - `true` if the application???including all of its windows???is hidden (e.g. with `Command-H`), `false` otherwise.
```

Comment:
All these charset formatting issues need to be reverted across all the docs that your editor changed

Reply:
I've reverted charset changes, see commit https://github.com/electron/electron/commit/88f173b1bf6a883f0de2c06c1eeb74942a641bb0